### DISCUSSION TITLE
Set a maximum number of lines shown on Code Blocks

### DISCUSSION THREAD

**Discussion by caffeinewriter** [outline/outline#7828] [Ideas]
Having code embedded in documentation is extremely nice without having to rely on an external provider like Github Gists or Gitlab Snippets.

However, for larger code blocks, it would be nice to have the option to set a maximum number of lines shown at a time, with the rest either collapsed, or scrollable, or just in general, to be able to set a max height for the block.

**Comment by tommoor** [comment]
Totally, I just created an issue for this https://github.com/outline/outline/issues/7829
